src/phoenix/session/session.py

```diff
@@ -85,7 +85,10 @@ def __init__(
             else None
         )
 
-        self.traces = Traces(trace_dataset.dataframe) if trace_dataset is not None else None
+        self.traces = Traces()
+        if trace_dataset:
+            for span in trace_dataset.to_spans():
+                self.traces.put(span)
```

Comment:
we could, but it's not adding a lot of value and i don't want to create two ways (and more code) for doing the same thing.

Reply:
It looks like the `Traces` class is already implemented to accept a list of spans to the initializer.

Reply:
Oh you're right. I was going to take it out but forgot..I'll just leave it then, since it's probably an intuitive expectation to have it as an option at `__init__` after all.